Client/Frontend/Browser/Tabs/TabTrayV2ViewModel.swift

```diff
@@ -43,27 +43,23 @@ class TabTrayV2ViewModel: NSObject {
     
         for (section, list) in dataStore {
             let sorted = list.sorted {
-                let firstTab = $0.lastExecutedTime ?? 0
-                let secondTab = $1.lastExecutedTime ?? 0
+                let firstTab = $0.lastExecutedTime ?? $0.sessionData?.lastUsedTime ?? 0
```

Comment:
During my investigation it seems that lastExecutedTime is set in memory when the tab is shown / selected which then also updated the lastUsedTime for its session. However, there is a case when the tab is not yet selected but just restored from the session (initial launch). This is why at time value of `lastExecutedTime` is there for the selected first tab but not for the non-selected tabs. Hence why checking both is the better option for just showing the tabs.